### PR TITLE
Feature/les 69 add basic internalization

### PR DESCRIPTION
**Pull request by DimitarShemet**
added internationalization for header

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,18 +3,26 @@ import { BrowserRouter, Routes, Route} from "react-router-dom";
...
+import {IntlProvider } from 'react-intl'; 
+import TranslationHelpers from './components/translations/translationHelpers'
+
+function App(): JSX.Element {
+  const [languageCode, setLanguageCode] = useState(TranslationHelpers.getCurrentLanguageCode())
+  const messages = TranslationHelpers.getLanguageMessages(languageCode)
 
-function App(): any {
   return (
+    <IntlProvider locale={languageCode} messages={messages}>
```
Comment (Terapool) [inline src/App.tsx:15]: Please use proper indentation for JSX hierarchy of elements

File: src/components/translations/en.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
+export let messages_en={
```
Comment (Terapool) [inline src/components/translations/en.js:1]: Please, separate '=' with spaces:

messages_en = {
messages_ru = {

File: src/components/body/Body.tsx [hunk 3]
```diff
@@ -2,10 +2,14 @@ import Main from "./content/Main";
 import Header from "./header/Header";
 import Footer from "./footer/Footer";
 
-const Body = (props: any) => {
+type BodyProps = {
+  onLanguageSwitch: (arg: string) => void
+}
+
+const Body = ({onLanguageSwitch}:BodyProps) => {
```
Comment (Terapool) [inline src/components/body/Body.tsx:9]: Space:
({onLanguageSwitch}: BodyProps)

File: src/components/body/header/Header.tsx [hunk 4]
```diff
@@ -1,32 +1,59 @@
...
       <Link to="/">
-        <div>Lessoner</div>
+        <div>
+          <FormattedMessage id="app.name"/> 
+      </div>
       </Link>
       <Link to="/categories">
-        <div >Categories</div>
+        <div>
+        <FormattedMessage id="app.categories"/>
```
Comment (Terapool) [inline src/components/body/header/Header.tsx:22]: Formatting indentation

**Review by Evgeniy121212 — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by OxygenO22 — APPROVED** [review]

**Pull request merged**